packages/core/src/Tab/TabChip.tsx

```diff
@@ -0,0 +1,27 @@
+import { Flex } from '../Flex'
+import { Text } from '../Text'
+import React from 'react'
+import { TabTriggerChip } from './Tab.styled'
+import { Box } from '../Box'
+
+export const TabChip = ({ hasHover, index, tab, value }) => {
+  return (
+    <TabTriggerChip
+      asChild
+      hover={hasHover}
+      value={`${value}-tab${index + 1}`}
+      key={`${value}-tab${index + 1}`}
+    >
+      <Flex alignItems='center' mr={2}>
+        {tab.icon && (
+          <Box ml={2} size={24}>
+            {tab.icon}
+          </Box>
+        )}
+        <Text py='5px' px={2} textStyle='paragraph' style={{ fontSize: 'Montserrat' }}>
```

Comment:
`style={{ fontSize: 'Montserrat' }}` ?

Reply:
the textStyle prop was not taking the styling. I was also having this issue with themeGet for BorderRadii cc: @craigpalermo , Happy to sync on it and try to diagnose the issue.